staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -59,6 +60,10 @@ func (d authenticatedDataString) AuthenticatedData() []byte {
 
 var _ value.Context = authenticatedDataString("")
 
+// LeaseReuseLength defines the period of time in seconds that each lease is
+// reused. We use var instead of const for testing purposes.
+var LeaseReuseLength int64 = 60
```

Comment:
Since we use in other places like testSetup, would a more explicit name like `LeaseReuseDurationSeconds` be worth it?

Reply:
done